src/SearchParameters/index.js

```diff
@@ -25,17 +19,38 @@ var filterState = require('./filterState');
 
 var RefinementList = require('./RefinementList');
 
+function objectHasKeys(obj) {
+  return Object.keys(obj).length > 0;
+}
```

Comment:
IMHO it could be moved to an external file in `functions` so it could be used in multiple files without the need to duplicate it. Same for `isEqualNumArr`.